pages/api/_utils.ts

```diff
@@ -0,0 +1,71 @@
+import { ConfigHelper, Config, LoggerInstance } from '@oceanprotocol/lib'
+import Web3 from 'web3'
+import ENS, { getEnsAddress as getEnsAddressVendor } from '@ensdomains/ensjs'
+import {
+  createClient,
+  dedupExchange,
+  TypedDocumentNode,
+  OperationContext,
+  fetchExchange
+} from 'urql'
+import { refocusExchange } from '@urql/exchange-refocus'
+
+let ens: any
+
+function getOceanConfig(network: string | number): Config {
```

Comment:
This is overengineered. ENS only works on ETH mainnet. So we know we only have to use the mainnet Infura URL. Whioch we simply define as a `const` in the below `getDummyWeb3()`, which itself should rather be named `getWeb3()`

Reply:
This is still over engineered, you are loading all of ocean.js to get a widely known Infura URL. Remove the need to have ocean.js as dependency